docs/source/recipes/fiftyone_terminology.rst

```diff
@@ -0,0 +1,123 @@
+.. _terminology-cheat-sheet:
+
+FiftyOne terminology
+====================
+
+.. default-role:: code
+
+A cheat sheet introducing you to the key terminology in the world of FiftyOne!
+
+Core components
+_______________
+
+.. list-table::
+   :widths: 20 80
+
+   * - **FiftyOne**
+     - The :ref:`open-source framework <fiftyone-library>`, the core library,
```

Comment:
I found that the table cells will automatically line wrap; no need to prepend `|` and manually control the line breaks!

Reply:
Oh nice!